<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialPage</class>
 <widget class="QDialog" name="InitialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ptn_yes">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化页面</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器IP地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_ip_address">
      <property name="text">
       <string>112.74.57.177</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_port">
      <property name="text">
       <string>12000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>目标温度上限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="max_target_temp">
      <property name="value">
       <number>28</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>目标温度下限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="min_target_temp">
      <property name="value">
       <number>18</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>中风速费率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="middle_fee_rate">
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>元/度</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>低风速费率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="low_fee_rate">
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>元/度</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>默认目标温度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="default_target_temp">
      <property name="value">
       <number>25</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>工作模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="mode">
      <item>
       <property name="text">
        <string>制冷</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>制热</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>高风速费率</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="high_fee_rate">
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>元/度</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>默认风速</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="default_speed">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>低</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>高</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
